tests/unit/macros/test_adapters_macros.py

```diff
@@ -213,21 +212,6 @@ def setUp(self):
 
         self.relation = DatabricksRelation.from_dict(data)
 
-    def __run_macro2(self, template, name, relation, *args):
```

Comment:
Was this code being used anywhere? Or is this just cleaning up unused cruft?

 I see its the same as `_run_macro` but I don't see that any references to `__run_macro2` were removed in this PR.

Reply:
I missed it in the refactor PR I sent out.  Unused cruft at this point.